expression/builtin_like_test.go

```diff
@@ -36,13 +36,17 @@ func (s *testEvaluatorSuite) TestLike(c *C) {
 		{"aAb", `Aa%`, 0},
 		{"aAb", "aA_", 1},
 	}
-	for _, tt := range tests {
-		fc := funcs[ast.Like]
-		f, err := fc.getFunction(s.ctx, s.datumsToConstants(types.MakeDatums(tt.input, tt.pattern, 0)))
-		c.Assert(err, IsNil)
-		r, err := evalBuiltinFunc(f, chunk.Row{})
-		c.Assert(err, IsNil)
-		c.Assert(r, testutil.DatumEquals, types.NewDatum(tt.match))
+	for i := 0; i < 10; i++ {
+		go func() {
+			for _, tt := range tests {
+				fc := funcs[ast.Like]
+				f, err := fc.getFunction(s.ctx, s.datumsToConstants(types.MakeDatums(tt.input, tt.pattern, 0)))
+				c.Assert(err, IsNil)
+				r, err := evalBuiltinFunc(f, chunk.Row{})
+				c.Assert(err, IsNil)
+				c.Assert(r, testutil.DatumEquals, types.NewDatum(tt.match))
+			}
+		}()
```

Comment:
The test should wait for all goroutines done with their works.
You may use `sync.WaitGroup`.

Reply:
added